flight/mcu/lib/teensyShared/servoControl.cpp

```diff
@@ -43,27 +49,26 @@ void round_off_angle(int& value) {
  * @param arg A struct containing pointers to objects needed to run the thread.
  *
  */
-void servoTickFunction(pointers* pointer_struct, PWMServo* servo_cw,
+void ServoControl::servoTickFunction(pointers* pointer_struct, PWMServo* servo_cw,
```

Comment:
since the necessary pointers are extracted from `pointer_struct` when this class is initialized, we don't have to pass it into this function anymore! Lets remove the argument from the function.